pandas/core/generic.py

```diff
@@ -3915,6 +3916,11 @@ def _take(
                 and is_range_indexer(indices, len(self))
             ):
                 return self.copy(deep=None)
+        else:
+            # We can get here with a slice via DataFrame.__geittem__
```

Comment:
Typo in getitem

Is this a column or row-based slice?

Reply:
column

Reply:
mind if i add the typo fix to my next Assorted CLN branch?

Reply:
id like to add a check for slice in `__getitem__` before we get here to return `self._slice(indexer)` instead, but that may (probably should) cause us to return a view instead of copy.

Once that is changed, I'd like to deprecate allowing slice in DataFrame.take (which we don't allow in Series.take).  It isn't documented, but still.

Reply:
Yeah fine by me.

Take returns a view for CoW even if it’s not a slice iirc. deprecating sounds good though.